### ISSUE TITLE
Build fail / Missing file: home_indicator-Swift.h

### ISSUE THREAD

**Issue by ghost** [RainwayApp/flutter_home_indicator#14]
Hi @lynn @cristian1980 @nohli  - thanks so much for making this plugin!
There is a missing header file called at line 2 of `HomeindicatorPlugin.m`:
`#import <home_indicator-Swift.h>`

In its current 2.0.1 version, any build with the plugin will fail. Could you please help provide the file? 
That would be awesome! 😊

**Comment by nohli** [comment]
@MoonshotQuest use this in your pubspec.yaml:

```
  home_indicator:
    git:
      url: https://github.com/nohli/flutter_home_indicator.git
      ref: fix-build
```

**Comment by ghost** [comment]
Thanks so much!
It builds now by replacing `HomeIndicatorPlugin.m` from your fork into the original one, or just use your `https://github.com/nohli/flutter_home_indicator.git` like you mentioned. :)

**Comment by lynn** [comment]
I just published 2.0.2 which contains @nohli's fix. Thank you @nohli !
